src/ui/qml/MainToolBar.qml

```diff
@@ -63,9 +79,9 @@ ToolBar {
 
         ToolButton {
             action: preferencesAction
-            iconSource: "qrc:/ui/icons/gear.png"
             implicitHeight: 24
-            implicitWidth: 24
+            style: iconButtonStyle
+            text: "\ue803"
```

Comment:
a "// this is X" at the end might not go astray on these, for those of us that aren't unicode-brained. sure, it's somewhat obvious if you read the context, but for scanning, it helps.